### PR TITLE
schettn-patch-2 to master (Update: README.md)

### PR DESCRIPTION
**Pull request by schettn**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [x] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
 - [x] Documentation (non-breaking change which adds documentation)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
-  The readme does not include usage and extend sections

**What is the new behavior (if this is a feature change)?**
- Now the readme includes usage and extend sections.

**Other information**:
-  🐍

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,80 +25,234 @@ The client unifies the way external data is handled and unites it under a clean,
...
 
-We do not have any external contributors yet, but if you want your name to be here, feel free
-to [contribute to our project](#contributing).
 
-## [](#copyright-and-license)Copyright and license
+/* Custom Client*/
+class CustomClient extends Client {
+  gql: ApolloEndpoint;
+  template: IMainTemplate;
+  session: SnekSession;
```
Comment (kleberbaum) [inline README.md]: Shouldn't this be a CustomSession?
```suggestion
  session: CustomSession;
```

**Review by kleberbaum — CHANGES_REQUESTED** [review]
Should be ok to merge after a few changes.

File: README.md [hunk 2]
```diff
@@ -25,80 +25,234 @@ The client unifies the way external data is handled and unites it under a clean,
...
 
-## [](#thanks)Thanks
+  //> Methods
+}
 
-We do not have any external contributors yet, but if you want your name to be here, feel free
-to [contribute to our project](#contributing).
 
-## [](#copyright-and-license)Copyright and license
+/* Custom Client*/
```
Comment (pinterid) [inline README.md]:
```suggestion
/* Custom Client */
```

File: README.md [hunk 3]
```diff
@@ -25,80 +25,234 @@ The client unifies the way external data is handled and unites it under a clean,
...
+
+## [](#thanks)Thanks
+We do not have any external contributors yet, but if you want your name to be here, feel free to [contribute to our project](#contributing).
+
+## [](#copyright-and-license)Copyright and license
 ![GitHub repository license](https://img.shields.io/badge/license-EUPL--1.2-blue)
 
 SPDX-License-Identifier: (EUPL-1.2)
+Copyright © 2019-2020 Simon Prast
 Copyright © Simon Prast
```
Comment (pinterid) [inline README.md]:
```suggestion
```

**Review by pinterid — CHANGES_REQUESTED** [review]
Some changes have to be made.

**Review by pinterid — APPROVED** [review]
Looks good.

**Review by kleberbaum — APPROVED** [review]
Should be ok to merge.

**Pull request merged**